File: trail1/공약수의 유무/presence-or-absence-of-a-common-divisor.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        bool flag = false;
        for (int i=A; i<=B; i++) {
            if (1920 % i == 0 && 2880 % i == 0) {
                flag = true;
                break;
            }
        }
        int val = flag==true ? 1 : 0;
        Console.WriteLine(val);
    }
}

File: trail1/공약수의 유무/presence-or-absence-of-a-common-divisor.py
lst = list(map(int, input().split()))
flag = False
A, B = lst[0], lst[1]
for i in range(A, B+1):
    if 1920 % i == 0 and 2880 % i == 0:
        flag = True
        break
print(1 if flag==True else 0) 